src/Generator/ActiveRecord/Command.php

```diff
@@ -24,7 +24,7 @@ public function __construct(
         private readonly string $namespace = 'App\\Model',
         #[Required]
         #[Regex(
-            pattern: '/^[a-z_][a-z0-9_]*$/i',
+            pattern: '/^[\w-]+(?:\.[\w-]+)?$/i',
```

Comment:
Can you add tests?

Reply:
I didn't add it as the test environment isn't set up yet in the repo.
An outside test can be seen here: https://regex101.com/r/aLjS42/1

Reply:
Actually there are some bugs: https://regex101.com/r/KX8vqz/1

Reply:
Maybe kind of this https://regex101.com/r/B8gCXv/1

Reply:
Also may be several dots. For example, in MSSQL: `database_name.schema_name.table_name`.

The table name specification is needed.

Reply:
I suggest to check only symbols, for example, `/^[\w-"`]+$/`. 
And do a more detailed check later.

Reply:
> I suggest to check only symbols, for example, `/^[\w-"`]+$/`. And do a more detailed check later.

I'm also of the same opinion

Reply:
Let me know which one I should use.
```
^(["'`]?)([\w][\w-]*)\1(?:\.((?1))(?2)\3)*$
```
https://regex101.com/r/3zbKmI/1

OR
```
^[\w\-."`]+$
```
https://regex101.com/r/3b3NYH/1

Reply:
afaik `-` is not permitted to use, isn't it?

Reply:
https://dev.mysql.com/doc/refman/8.0/en/identifiers.html
tested in MySql, it's permitted